src/core/storage/secureStorage/secureStorage.ts

```diff
@@ -6,8 +6,11 @@ import {
 enum KeyStoreKeys {
   APP_PASSCODE = "app-login-passcode",
   APP_OP_PASSWORD = "app-operations-password",
+  PASSWORD_SKIPPED = "app-password-skip",
```

Comment:
These are not meant to be going into the key store. This needs to be changed.

The key store is for secrets. There is a mechanism in the core for storing misc items like this called basic storage.

Reply:
@bao-sotatek can support here